### DISCUSSION TITLE
Where to find download bc 2021 wave 2 19.1 thanks

### DISCUSSION THREAD

**Discussion by Master-Manager** [microsoft/ALAppExtensions#15258] [General] [answered]
Where to find download bc 2021 wave 2 19.1 thanks

**Comment by SimonOfHH** [comment, marked as answer]
According to [Docs](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/deployment/update-versions-19#available-updates-for-business-central-2021-release-wave-2) it's not yet available for download.
